src/control/server/ioserver/config.go

```diff
@@ -186,6 +187,20 @@ func (c *Config) CmdLineEnv() ([]string, error) {
 		return nil, err
 	}
 
+	// Provide special handling for the ofi+verbs provider.
+	// When specified, perform a special lookup to find the alias
+	// of the given fabric_iface.  Then set a new environment variable "OFI_DOMAIN"
+	// to the name of the device alias.  For example, a fabric_iface of "ib0"
+	// has a device alias of "hfi1_0", and "OFI_DOMAIN=hfi1_0" will be added
+	// to this I/O server's environment
```

Comment:
Comment feels a little like it's just narrating the (already clear and readable) code below. In my view it would be more useful for readers for the comment to explain why the env var is needed to begin with, and why the env var needs the alias.

Reply:
I'll rework it with a shorter comment that tries to address why we are doing this.   The intention of the comment was to show what an alias was and point out what the new environment variable would look like since you do not get a sense of device names like ib0 and the alias hfi1_0 from reading the code alone.  I am glad the code itself is clear though.  That's a good start.